tokio/src/runtime/handle.rs

```diff
@@ -93,6 +96,13 @@ impl Handle {
     /// let handle = Handle::current();
     /// handle.spawn(async {
     ///     println!("now running in the existing Runtime");
+    /// });
+    ///
+    /// thread::spawn(move || {
+    ///     // Notice that the handle is created outside of this thread and then moved in
+    ///     handle.block_on(async { /* ... */ })
+    ///     // This next line would cause a panic
+    ///     // let handle2 = Handle::current();
     /// })
```

Comment:
I like to join threads in our tests to ensure they eventually exit and don't panic.
```suggestion
    ///
    /// # let handle = 
    /// thread::spawn(move || {
    ///     // Notice that the handle is created outside of this thread and then moved in
    ///     handle.block_on(async { /* ... */ })
    ///     // This next line would cause a panic
    ///     // let handle2 = Handle::current();
    /// });
    /// # handle.join().unwrap();
```